src/index.tsx

```diff
@@ -117,18 +119,20 @@ ReactDOM.render(
     <Provider store={store}>
         <TopRightControls/>
         <Router>
-            <Switch>
-                <Route path="/draft/:id" component={Draft}/>
-                <Route exact path="/" component={Menu}/>
-                <Route path="/presets" component={Menu}/>
-                <Route path="/preset/create" component={Menu}/>
-                <Route path="/preset/:id" component={Menu}/>
-                <Route path="/preset/:id/new" component={Menu}/>
-                <Route path="/spectate/:id" component={SpectateDraft}/>
-                <Route path="/spectate" component={Menu}/>
-                <Route path="/practice" component={Menu}/>
-                <Route component={NotFound404}/>
-            </Switch>
+            <ScrollToTop>
```

Comment:
Is that one really necessary? It somehow feels nicer if the page does not jump back to the top when changing tabs…

Reply:
I feel that is the user expected behaviour when you navigate to a different page. If we remove this code and navigate to spectate draft that is listed, for example, the user has scroll manually to the top. This is not happening in the current production version, but happens in the latest version. 

There is an alternative way to make the scroll animation smoother.

Reply:
Can we somehow get different behaviour for clicking on tabs and the other page transitions? :thinking:

Reply:
It seems difficult at this point. I can either remove that functionality for now or we try to address selective scroll to top in subsequent updates.

Reply:
Let's keep the automatic scroll to top functionality out for now